exercises/grade-school/src/lib.rs

```diff
@@ -0,0 +1,22 @@
+#[allow(unused_variables)]
+
+pub struct School {
+}
+
+impl School {
+    pub fn new() -> School {
+        unimplemented!()
+    }
+
+    pub fn add(&mut self, grade: u32, student: &str) {
+        unimplemented!()
+    }
+
+    pub fn grades(&self) -> Vec<u32> {
+        unimplemented!()
+    }
+
+    pub fn grade(&self, grade: u32) -> Option<Vec<String>> {
```

Comment:
https://github.com/exercism/xrust/issues/254#issuecomment-276057035

> It would be nice if, when a constraining choice like this is made, the README or stub file could be explicit about why that interface was chosen.

I failed to do this. It could be something like "If grade returned an `Option<&Vec<String>>`, the internal implementation would be forced to keep a `Vec<String>` to lend out. By returning an owned vector instead, the internal implementation is free to use whatever it chooses."